src/playground/views.clj

```diff
@@ -3,7 +3,8 @@
    [hiccup.page :as page]
    [hiccup.table :as table]
    [playground.services.invoices.retrieve-all.endpoint :as retrieve-all]
-   [playground.services.invoices.retrieve.endpoint :as retrieve]))
+   [playground.services.invoices.retrieve.endpoint :as retrieve]
```

Comment:
Once you address all other feedback, and you are sure that everything works correctly, as a last commit split this ns across the "services".

Each service will have the following structure:

```
.
├── endpoint.clj
├── logic.clj
└── views.clj
```